scripts/birds.js

```diff
@@ -1,59 +1,161 @@
     lifespan: "41 years",
     group: "bird",
     image: "../images/transparent-img/yellow-tailed-black-cockatoo.PNG",
     food: "Fruit, seeds and other plant material",
-    shortDescription:   `It has a short crest on the top of its head. Its plumage is mostly brownish black and 
-                        it has prominent yellow cheek patches and a yellow tail band. The body feathers are edged 
+    shortDescription:   `It has a short crest on the top of its head. Its plumage is mostly brownish black and
+                        it has prominent yellow cheek patches and a yellow tail band. The body feathers are edged
                         with yellow giving a scalloped appearance.`,
-    fulldescription:    `The adult male has a black beak and pinkish-red 
-                        eye-rings, and the female has a bone-coloured beak and grey eye-rings. In flight, yellow-tailed 
-                        black cockatoos flap deeply and slowly, with a peculiar heavy fluid motion. Their loud, 
-                        wailing calls carry for long distances. The whiteae is found south of Victoria to the East 
-                        of South Australia and is smaller in size. The yellow-tailed black cockatoo is found in 
-                        temperate forests and forested areas across south and central eastern Queensland to 
+    fulldescription:    `It has a short crest on the top of its head. Its plumage is mostly brownish black and
+                        it has prominent yellow cheek patches and a yellow tail band. The body feathers are edged
+                        with yellow giving a scalloped appearance. The adult male has a black beak and pinkish-red
+                        eye-rings, and the female has a bone-coloured beak and grey eye-rings. In flight, yellow-tailed
+                        black cockatoos flap deeply and slowly, with a peculiar heavy fluid motion. Their loud,
+                        wailing calls carry for long distances. The whiteae is found south of Victoria to the East
+                        of South Australia and is smaller in size. The yellow-tailed black cockatoo is found in
+                        temperate forests and forested areas across south and central eastern Queensland to
                         southeastern South Australia, including a very small population persisting in the Eyre Peninsula.`,
     length: "65cm",
     weight: "900g",
     found: "Southeastern Australia"
-};
\ No newline at end of file
+};
+
+const addBirds = () => {
+    birds.push(bird1);
+    birds.push(bird2);
+    birds.push(bird3);
+}
+
+let animalMenuEl = document.querySelector(".animalmenu__cointainer")
+let birdFirstNameEl = document.querySelector(".section-2__title--changeContent-1")
+let birdLastNameEl = document.querySelector(".section-2__title--changeContent-2")
+let birdImgEl = document.querySelector(".section-2__img--changeContent")
+let birdDescEl = document.querySelector(".section-2__text--changeContent")
```

Comment:
I think adding semicolons is recommended for consistency and to avoid potential issues other than that everything looks fine